Scoop/View/SummaryInfoSection.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  PostRideSummaryInfoView.swift
+//  Scoop
+//
+//  Created by Reade Plunkett on 3/9/22.
+//
+
+import UIKit
+
+class SummaryInfoSection: UIView {
+    
+    private let infoLabel = UILabel()
+    private let infoImageView = UIImageView()
+    
+    private var title = ""
+    private var image: UIImage?
+    
+    init(title: String, image: UIImage?) {
+        super.init(frame: .zero)
+        self.title = title
+        self.image = image
```

Comment:
I think if you do `self.image = image` before the `super.init(frame: .zero)`, `image` doesn't have to be an optional.

Reply:
Since ImageViews take in Images as optionals, it's cleaner to leave everything as an optional instead of having to unwrap it to pass the value. Essentially we let the imageview do the unwrapping for us